### ISSUE TITLE
Issue linking with libfmt

### ISSUE THREAD

**Issue by jeefberkey** [Alexays/Waybar#89]
I'm making a new issue, because #73 has been closed for quite some time.

I have isolated the issue a bit more. I can build today and run waybar from /usr/local/bin if I use the older version of libfmt from the 0.1.3 tag.

```
$ git checkout master
$ meson build
$ ninja -C build
$ sudo ninja -C build install
$ waybar
waybar: symbol lookup error: waybar: undefined symbol: _ZN3fmt2v58internal10basic_dataIvE23ZERO_OR_POWERS_OF_10_32E
$ git checkout 0.1.3 subprojects/
$ meson build --reconfigure
$ sudo ninja -C build install
$ waybar
$ # runs fine
```

So it looks like the newer version of fmt or the new wrap is not working correctly.

**Comment by Alexays** [comment]
The fmt 5.2.1 fix this behaviour, and is just available on wrapdb 🎉

**Comment by Alexays** [comment]
Fixed in 2d2fb8804068770c604293e9128fc59b6b962e43

**Comment by jeefberkey** [comment]
:clap: thanks!
